store/postgres/src/lib.rs

```diff
@@ -10,19 +12,23 @@ extern crate fallible_iterator;
 extern crate futures;
 extern crate graph;
 extern crate graph_graphql;
+extern crate graphql_parser;
 extern crate inflector;
 extern crate lazy_static;
 extern crate lru_time_cache;
 extern crate postgres;
 extern crate serde;
 extern crate uuid;
 
+mod block_range;
 mod chain_head_listener;
 mod db_schema;
 mod entities;
 mod filter;
 mod functions;
 mod jsonb;
+mod mapping;
+mod mapping_sql;
```

Comment:
I don't think `mapping_sql` is a great name for this module, maybe a more describing name would be `relational_queries`. Also it'd be nice to start having comments above the modules explaining what they contain.